src/components/RenderRocket.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RenderRocket = (
+  {
+    name, desc, id, img,
+  },
+) => (
+  <li className="rocket-li">
+    <div className="img-container">
+      <img alt={name} src={img} />
+    </div>
+    <div className="rocket-info">
+      <h2>{name}</h2>
+      <p>{desc}</p>
+      <button type="button" id={id}>
+        Reserve rocket
+      </button>
+    </div>
+  </li>
+);
+
+RenderRocket.propTypes = {
+  name: PropTypes.string.isRequired,
+  desc: PropTypes.string.isRequired,
+  img: PropTypes.string.isRequired,
```

Comment:
There is a console error that states the value of ```img``` is an array when you required a string. This may cause issues in the future, so it would be better to fix it. 

![image](https://user-images.githubusercontent.com/74314917/156246342-e5f132ca-f981-466f-9be9-b3498960baaa.png)